opensearch_py_ml/operations.py

```diff
@@ -475,7 +475,7 @@ def _terms_aggs(
         except IndexError:
             name = None
 
-        return build_pd_series(results, name=name)
+        return build_pd_series(results, index_name=name, name="count")
```

Comment:
How come its using "count" here but before it was using name?

Reply:
In pandas 2.0.3, a change in the value_counts method resulted in the following behavior:

The method now uses "count" as the name for the values column, while the original column name (e.g., "Carrier") is used for the index name. This differs from earlier versions, where the values column would inherit the name of the original column.
<img width="375" alt="pd_153" src="https://github.com/user-attachments/assets/fd97983b-c583-49b5-bb58-45475af65b50">
<img width="360" alt="pd_203" src="https://github.com/user-attachments/assets/5093dfb5-ae4e-40f8-83e2-867c3be4f43b">

Reply:
Hey @dhrubo-os what are your thoughts? Thanks for the info @Yerzhaisang

Reply:
In pandas 1.5.3, the [series name](https://github.com/pandas-dev/pandas/blob/2e218d10984e9919f0296931d92ea851c6a6faf5/pandas/core/algorithms.py#L955) is used for the [result series](https://github.com/pandas-dev/pandas/blob/2e218d10984e9919f0296931d92ea851c6a6faf5/pandas/core/algorithms.py#L998).

In pandas 2.0.3, the [series name](https://github.com/pandas-dev/pandas/blob/0f437949513225922d851e9581723d82120684a6/pandas/core/algorithms.py#L835) is used for the [index name](https://github.com/pandas-dev/pandas/blob/0f437949513225922d851e9581723d82120684a6/pandas/core/algorithms.py#L894), while the [result series](https://github.com/pandas-dev/pandas/blob/0f437949513225922d851e9581723d82120684a6/pandas/core/algorithms.py#L896) name is set to [this one](https://github.com/pandas-dev/pandas/blob/0f437949513225922d851e9581723d82120684a6/pandas/core/algorithms.py#L836).